Doc/faq/general.rst

```diff
@@ -222,7 +222,7 @@ releases are announced on the comp.lang.python and comp.lang.python.announce
 newsgroups and on the Python home page at https://www.python.org/; an RSS feed of
 news is available.
 
-You can also access the development version of Python through Mercurial.  See
+You can also access the development version of Python through Git.  See
 https://docs.python.org/devguide/faq.html for details.
 
```

Comment:
Devguide URL should be updated to http://cpython-devguide.readthedocs.io.
Also, the FAQ has been removed from the Dev Guide, so the section you want is perhaps http://cpython-devguide.readthedocs.io/communication.html